Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -257,7 +257,11 @@ - (void)pause:(BOOL)pause controlSentToCall:(OCTCall *)call
             callToUpdate.pausedStatus |= OCTCallPausedStatusByUser;
         }
         else {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wassign-enum"
             callToUpdate.pausedStatus &= ~OCTCallPausedStatusByUser;
+#pragma clang diagnostic pop
+
```

Comment:
Tried type casting, but the warnings still show up here. Thus resorting to this instead.